README.md

```diff
@@ -18,7 +18,36 @@ This Digital Twin Definition Language (DTDL) Model Generator parses your DTDL js
 - *Namespace*: The namespace that will be injected into the generated model classes.
 - *JsonModelsDirectory*: The directory that contains the DTDL json files that we'll be parsing.
 - *OutputDirectory*: The directory that the generated model classes will be placed in.
-- *IncludeTemplateProject*: Whether or not to include the template project in the output directory.
+- *CopyrightHeader*: The header comment that contains copyright information.
+
+## Prerequisites
+The following are some Prerequisites/assumptions to be considered:
+
+For Developers
+- .Net 6 must be installed, while building the model generator locally. Backward compatibility is not supported.
+- The system should have GIT installed as the model generator uses MinVer package which requires GIT.
+
+For Users
+- The following classes have been copied at the location where model generator assembly is being executed:
+        "Extensions.cs",
+        "ModelHelper.cs",
+        "Relationship.cs",
+        "RelationshipCollection.cs",
+        "RelationshipEqualityComparer.cs",
+        "SourceValueAttribute.cs",
+        "TwinEqualityComparer.cs"
+- The ModelGeneratorOptions property "CopyrightHeader" should have "// " prefix.
+- The following NuGet packages references should be added: 
+		Azure.DigitalTwins.Core(1.2.2)
+		Microsoft.Azure.DigitalTwins.Parser(3.12.7)
+		MinVer(3.1.0)
+		StyleCop.Analyzers(1.1.118)
+		System.Text.RegularExpressions(4.3.1)
```

Comment:
I'm not sure about documenting these, since the packages (especially the version) could change and be out-of-sync with this readme. Additionally, installing the model generator NuGet package will automatically add these package references.

Reply:
ReadMe Updated.